### PR TITLE
szl 1.0.1 (new maintainer)

### PR DESCRIPTION
**Pull request by aecolley**
I fixed many bugs in szl, but Google has abandoned it and no longer
responds to bug reports or messages on szl-users. So I took it over.

I sent an earlier version of this formula to Adam Vandenberg, whom I found
in the git commit history. But he said "Please send this to the Homebrew
mailing list, not me directly; or open a ticket with this information. I
don't maintain this formula." I'll maintain it, then.

This formula is cleaner (no patches), more stable (the code is more
portable) and more useful (bugs are fixed) than the previous one.

### CODE REVIEW

File: Library/Formula/szl.rb [hunk 1]
```diff
@@ -1,79 +1,49 @@
...
-  # https://code.google.com/p/szl/issues/detail?id=34
-  patch :DATA
+  # We don't need gobjdump for core functionality, but only for
+  # "szl --print_code --native". We can use it if it's installed later.
+  depends_on "binutils" => :optional
 
   def install
-    ENV["OBJDUMP"] = "#{HOMEBREW_PREFIX}/bin/gobjdump"
+    if build.head?
+      system "./autogen.sh", "-f"
```
Comment (dunn) [inline Library/Formula/szl.rb]: This can be just one line `system "./autogen.sh", "-f" if build.head?`

**Comment by DomT4** [comment]
We're usually *very* reluctant to accept personal forks of a project unless upstream have signed off on the fork as the new development location. That's unlikely to happen with Google, which could be tricky.

We tend to be reluctant because it can surprise users to discover they thought they had one thing installed and now have a completely different fork installed with none of the original developers/etc.

Will leave it open for other owners here to review/discuss. If we choose to accept this it may end up going in under a new name rather than taking over the old one.

**Comment by mikemcquaid** [comment]
> We're usually very reluctant to accept personal forks of a project unless upstream have signed off on the fork as the new development location. That's unlikely to happen with Google, which could be tricky.

That's the case here, I'm afraid.

**Comment by aecolley** [comment]
That's understandable. I'd have the same response if I were in your position. I thank you for providing the facility of Taps, which makes installing my szl almost as easy as if you'd accepted this pull request.

**Comment by DomT4** [comment]
Thanks for understanding, We appreciate that.

**Pull request closed without merging**